From c24da36d8aef2ff74f4a60e9cdf5bb5473eb77bf Mon Sep 17 00:00:00 2001
From: Alexander Martinz <amartinz@shiftphones.com>
Date: Tue, 26 Feb 2019 20:54:02 +0100
Subject: [PATCH 10/20] AndroidManifest: add Foreground Service permission

This is required since API 28 to be able to start services
in foreground.

Change-Id: Iae6dc836cfee9d1fc0231f54bb878a2b78dbb15e
Signed-off-by: Alexander Martinz <amartinz@shiftphones.com>
---
 AndroidManifest-gradle.xml | 3 +++
 AndroidManifest.xml        | 3 +++
 2 files changed, 6 insertions(+)

diff --git a/AndroidManifest-gradle.xml b/AndroidManifest-gradle.xml
index 2af3270..374e80d 100644
--- a/AndroidManifest-gradle.xml
+++ b/AndroidManifest-gradle.xml
@@ -45,6 +45,9 @@
     <uses-permission android:name="android.permission.RECORD_AUDIO" />
     <uses-permission android:name="android.permission.MODIFY_AUDIO_SETTINGS" />
 
+    <!-- Allows Eleven to start services in foreground -->
+    <uses-permission android:name="android.permission.FOREGROUND_SERVICE" />
+
     <!-- Accelerometer feature for shake to play -->
     <uses-feature android:name="android.hardware.sensor.accelerometer" />
 
diff --git a/AndroidManifest.xml b/AndroidManifest.xml
index e786c9c..af44122 100644
--- a/AndroidManifest.xml
+++ b/AndroidManifest.xml
@@ -50,6 +50,9 @@
     <uses-permission android:name="android.permission.RECORD_AUDIO" />
     <uses-permission android:name="android.permission.MODIFY_AUDIO_SETTINGS" />
 
+    <!-- Allows Eleven to start services in foreground -->
+    <uses-permission android:name="android.permission.FOREGROUND_SERVICE" />
+
     <!-- Accelerometer feature for shake to play -->
     <uses-feature android:name="android.hardware.sensor.accelerometer" />
 
-- 
2.17.1

